devtools/pgdump-lite/dumpdata.go

```diff
@@ -152,11 +152,11 @@ func DumpData(ctx context.Context, conn *pgx.Conn, out io.Writer, skip []string)
 			}
 			for i, v := range vals {
 				if i > 0 {
-					io.WriteString(out, "\t")
+					_, _ = io.WriteString(out, "\t")
 				}
-				io.WriteString(out, string(*v.(*scannable)))
+				_, _ = io.WriteString(out, string(*v.(*scannable)))
 			}
-			io.WriteString(out, "\n")
+			_, _ = io.WriteString(out, "\n")
```

Comment:
If we fail to write to the output file, we want the operation to fail immediately with the proper error message. Otherwise, we'll continue dumping data and writing blindly to a broken pipe with no indication of a problem.

This is a legitimate case where the linter would have found a bug (errors that shouldn't be ignored); your original code returning the error for these was the proper fix.

Reply:
sounds good.  I'll put the panic back in for this section of code.